src/__tests__/components/ShoppingCart/ShoppingCartContainer.test.tsx

```diff
@@ -0,0 +1,35 @@
+import ShoppingCartContainer from "@/components/ShoppingCart/ShoppingCartContainer";
+import renderWithMockedProvider from "../../utils";
+import { render, screen, fireEvent, within } from "@testing-library/react";
+import { mockCartData } from "@/components/MockData/MockCartData";
+
+describe("ShoppingCart component", () => {
+  test("Should render checkout button", () => {
+    renderWithMockedProvider(<ShoppingCartContainer userShoppingCart={[]} />);
```

Comment:
Could you please explain the reason for using mock data? Thank you!

Reply:
The database has no user shopping cart information available for the user shopping cart page. Without using the mock data the shopping cart component will always show an empty list. The mock data is used in the test.